### PR TITLE
[CW] [105350578] MarkerWithLabel

### PR DESCRIPTION
**Pull request by c0**
- Pass in `@attr.markerLabelOptions` which internally uses `MarkerWithLabel`.
- Add option to `data/listings` to limit number of pins. `@attr.pinLimit`

[Story](https://www.pivotaltracker.com/story/show/105350578)
- [x] Specs pass

### CODE REVIEW

**Comment by warrenv** [comment]
Can you discuss the changes to the gulp config?

**Comment by c0** [comment]
`MarkerWithLabel` isn't an AMD component and adds to the global scope. The changes in gulp are to automatically wrap it (rather than editing the source).

That being said, there's an issue with AG's tests failing because this file is looking for a global `google`. I'll close this until I figure out how/where to fix this.

**Comment by c0** [comment]
PR updated and properly defers initializing `MarkerWithLabel` until it's needed.

File: test/spec/components/ui/markers_spec.coffee [hunk 1]
```diff
@@ -87,6 +92,57 @@ define [], () ->
...
+          expect(@marker.constructor).toEqual google.maps.Marker
+
+      describe 'with label options', ->
+        beforeEach ->
+          @setupComponent
+            markerLabelOptions: (datum) ->
+              content: datum.price
+          @marker = @component.createMarker(datum)
+
+        it 'is a standard marker type', ->
```
Comment (sbrauer) [inline test/spec/components/ui/markers_spec.coffee]: Is the `it` string what you intended?
It's the same as line 101, but the expectation is different.

Comment (c0) [inline test/spec/components/ui/markers_spec.coffee]: Good catch. I updated the copy. The assertion is correct, but not the description.

**Comment by sbrauer** [comment]
:+1:

**Pull request merged**